### ISSUE TITLE
d3 v4 not compatible with c3 integration in pivottable

### ISSUE THREAD

**Issue by tsando** [nicolaskruchten/pivottable#693]
Referring to this example to add C3 chart renderers: http://nicolas.kruchten.com/pivottable/examples/c3.html

I can't get it to work using the latest stable version of d3 (https://d3js.org/d3.v4.min.js). It only works using the v3 version. The problem is that the v4 of d3 has a lot of features which aren't available in v3. 

Do you know if there is a work-around for this issue?

Thanks.

**Comment by nicolaskruchten** [comment]
Yes, unfortunately C3js doesn't support D3 version 4 at the moment: https://github.com/c3js/c3/issues/1648

The only workaround would be to use a different charting plugin like the Google Charts one.

**Comment by paulovieira** [comment]
Might be worth to check billboard.js: https://naver.github.io/billboard.js/

It's a fork of C3 based on D3 version 4. I think the API/options to create the charts are essentially the same.

**Comment by nicolaskruchten** [comment]
@paulovieira interesting, I hadn't heard about billboard.js. If the API is similar enough, perhaps just loading it as the `c3` object would work?

I'm slowly moving to supporting plotly.js as the preferred charting renderer for this library (given that I'm a Plotly employee now!) so I don't have a lot of time/incentive to devote to C3/Billboard/etc.
